.github/actions/build/action.yml

```diff
@@ -0,0 +1,27 @@
+name: Build and Test
+description: Runs build
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install pulumictl
+      uses: jaxxstorm/action-install-gh-release@v1.10.0
+      with:
+        repo: pulumi/pulumictl
+    - name: Setup Node
+      uses: actions/setup-node@v4
+      with:
+        node-version: 18.19.x
```

Comment:
do we not want to get new minor version automatically? At least that's what we do for most of the other projects